diff -rupN nss-3.14.1/mozilla/security/nss/cmd/manifest.mn ournss/mozilla/security/nss/cmd/manifest.mn
--- nss-3.14.1/mozilla/security/nss/cmd/manifest.mn	2012-11-17 03:52:36.000000000 -0800
+++ ournss/mozilla/security/nss/cmd/manifest.mn	2013-01-23 11:37:10.848557142 -0800
@@ -46,7 +46,6 @@ DIRS = lib  \
  selfserv  \
  signtool \
  signver \
- shlibsign \
  smimetools  \
  ssltap  \
  strsclnt \
